Casks/pikachuexe-freetube.rb

```diff
@@ -1,13 +1,19 @@
-cask "freetube" do
-  # Change this when release contains arm64 version
-  arch arm: "x64", intel: "x64"
+cask "pikachuexe-freetube" do
+  arch arm: "arm64", intel: "x64"
```

Comment:
I intentionally make it `x64` for arm to use x64 version for 0.19.1
It should only be change to arm64 after a arm64 version is built (and URL would be changed at the same time)

No idea why we change now before it's even available (at most this can be  a draft PR)

Reply:
That's fine with me. I am only trying to help get this ready. No one is actually supposed to use this tap until there is an arm64 build available, right? So I showed the structure, e.g., for the different download urls. It's up to you to decide what you want to do and when.

I'm not sure how much sense it makes to include Intel in this tap at all. Its only purpose could be to provide the arm64 build and let Intel users stick with homebrew-cask, in which case some of this would have to be changed anyway.

Reply:
I guess we can make the cask to depend on arm (so intel mac can't install it) and remove intel version
Care to do it in this PR too?

Reply:
Sure. Pushed a second commit.

I'll try to take a look at the automation workflows you asked about in the FreeTube issue, too. But time is limited, so it might take a little while.